dspace/config/config-definition.xml

```diff
@@ -21,6 +28,9 @@
         <system/>
         <!-- Load Environment variables (does not automatically reload) -->
         <env/>
+        <!-- Load Environment variables that contain two underlines in their names, and exchange those underlines with a
+             dot, as Bash does not allow variables that contains dots in their names.-->
```

Comment:
This comment is now slightly inaccurate, as this has been extended to both periods and dashes.  Might want to restate this like:
`<!-- Load Environment variables that contain two underlines in their names. Some DSpace configurations use
special characters in their key (e.g. periods and dashes) which are not supported in Bash. This provider translates "___P___" to periods and "___D___" to dashes in environment variables. -->`